package/base-files/files/lib/preinit/10_indicate_preinit

```diff
@@ -96,8 +96,13 @@ preinit_config_board() {
 			json_select "$netdev"
 				json_get_vars path path
 				next_eth="$(echo "$netdev" | grep 'eth[0-9]*' | tr -dc '[0-9]')"
-				[ "$next_eth" -gt "$max_eth" ] && max_eth=$next_eth
-				if [ -n "$path" -a -h "/sys/class/net/$netdev" ]; then
+				if [ -n "${next_eth:-}" ] && \
+				   [ "${next_eth}" -gt "${max_eth}" ]; then
+					max_eth="${next_eth}"
+				fi
+				if [ -n "${path}" ] && \
```

Comment:
The `&&`'s don't TECHNICALLY need a `\` afterwards, since the shell interprets them, similarly to `|`s, as a request to continue the command on the next line. But this is fine.

Reply:
Yes, TECHNICALLY you are correct, however (and as this is a pattern I simply just always follow) I remember that there IS a difference in some cases (shell being shell). I don't mind digging around what the exact reason was. Could be that it was preferred in the google bash style guide?